jps-plugin/jps-plugin.iml

```diff
@@ -5,9 +5,8 @@
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
     </content>
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="IntelliJ IDEA IU-163.15188.11" jdkType="IDEA JDK" />
```

Comment:
Don't include this file.  This entry is specific to your machine, and should remain as "interitedJdk".  In general, don't check in any changes to the project or files in the .idea directories.  If you have general changes that you think would benefit everybody, then we can discuss those.